Obsidian/Net/Packets/Play/Serverbound/PlayerBlockPlacement.cs

```diff
@@ -37,7 +37,9 @@ public async ValueTask HandleAsync(Server server, Player player)
 
             var position = this.Position;
 
-            var interactedBlock = server.World.GetBlock(position);
+            var b = server.World.GetBlock(position);
```

Comment:
What return type does server.World.GetBlock have?

Reply:
GetBlock now returns a nullable block. This is to cast it from Block? to Block. [ref](https://github.com/ObsidianMC/Obsidian/pull/118/files/06ddba56639af82233220ad3a04be4312424b6f3#diff-07d01dc2121192313271a9b27d049be22f90141a5a9f21001f3839f2499ea90dR208)

Reply:
Then I'd work with the property values `b.HasValue` and `b.Value`, but it's up to you if you wanna do that.